yt/visualization/plot_container.py

```diff
@@ -857,19 +869,58 @@ def show_axes(self, field=None):
         return self
 
 
-class ImagePlotContainer(PlotContainer):
+class ImagePlotContainer(PlotContainer, abc.ABC):
     """A container for plots with colorbars."""
 
     _colorbar_valid = False
 
     def __init__(self, data_source, figure_size, fontsize):
         super().__init__(data_source, figure_size, fontsize)
-        self.plots = PlotDictionary(data_source)
         self._callbacks = []
-        self._cbar_minorticks = {}
-        self._background_color = PlotDictionary(self.data_source, lambda: "w")
         self._colorbar_label = PlotDictionary(self.data_source, lambda: None)
 
+    def _get_default_handlers(
+        self, field, default_display_units: Unit
```

Comment:
`_handlers.py` already uses both, I don't know how confusing it is in the current form.
Long term I think it'd be preferable to import unyt as a namespace (`import unyt as un`) in most places. In this particular module this would lead to a lot of churn because `un` is also used as a name variable, so I chose to keep it simple for now. WDYT ?

Reply:
> Long term I think it'd be preferable to import unyt as a namespace (import unyt as un) in most places.

Ya, I agree with that for sure, which would definitely have avoided my confusion here (which is also wrapped up in the question of importing from unit vs. yt.unit).